graph/api/api_test.go

```diff
@@ -1071,6 +1201,40 @@ func respond(w http.ResponseWriter, code int, payload interface{}) {
 	_, _ = w.Write(response)
 }
 
+// Helper method that tests the objects are equal and if they aren't will
+// unmarshal them into a json object and diff them. This way the output of the failure
+// is actually useful. Otherwise printing the byte slice results in completely incomprehensible output.
```

Comment:
```suggestion
// is actually useful. Otherwise printing the byte slice results is incomprehensible.
```

Reply:
Agreed it was incomprehensible, I always just ignored that part, clipped the whole differing json, and then diff'd it in git gui.  Thanks for taking time to improve.

Reply:
Ah I could've done that as well. I mostly didn't want to scroll through the large output. Hopefully won't need to update these expectation files much.